ajax/libs/ng-browser-info/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ng-browser-info",
+  "filename": "ngBrowserInfo.min.js",
+  "description": "AngularJS service that gives you a collection of methods for knowing more about your client browser ",
```

Comment:
The description need to as the same with upstream.
From `"AngularJS service that gives you a collection of methods for knowing more about your client browser "` to [`"ngBrowserInfo is an AngularJS service that gives you a collection of methods for knowing more about your client browser."`](https://github.com/transferwise/ng-browser-info/blob/master/package.json#L3)
Thank you.